cmd/lifecycle/detector.go

```diff
@@ -33,8 +34,8 @@ func (d *detectCmd) Init() {
 	cmd.FlagAppDir(&d.appDir)
 	cmd.FlagPlatformDir(&d.platformDir)
 	cmd.FlagOrderPath(&d.orderPath)
-	cmd.FlagGroupPath(&d.groupPath)
-	cmd.FlagPlanPath(&d.planPath)
+	cmd.FlagGroupPath(d.platformAPI, "", &d.groupPath)
+	cmd.FlagPlanPath(d.platformAPI, "", &d.planPath)
```

Comment:
Should we allow the user to set the `layers` directory in this command or just keep it empty as is now and then the default `groupPath` and `planPath` will be under `.`?